src/common/SurgeLogger.cpp

```diff
@@ -0,0 +1,57 @@
+#include "SurgeLogger.h"
+#include "spdlog/sinks/basic_file_sink.h"
+#include "spdlog/sinks/stdout_sinks.h"
+
+#include "spdlog/logger.h"
+#include "spdlog/async.h"
+#include <memory>
+
+
+typedef std::shared_ptr< spdlog::logger > log_t;
+
+log_t internal_surge_log_init()
+{
+  std::vector<spdlog::sink_ptr> sinks;
+  
+  /*
+  ** Where to put the logfile is actually OS dependent. 
+  ** The approach we take here is to add a pair of sinks
+  ** to the logger, the first to stderr
+  */
+  auto stderr_logger = std::make_shared<spdlog::sinks::stderr_sink_mt>();
+  sinks.push_back( stderr_logger );
```

Comment:
Is that a convention that is used widely in Surge i.e. extra space after `(` and before `)`? Also, I'm puzzled about variable naming i.e. you use underscores to separate words.